src/lib.rs

```diff
@@ -94,7 +94,12 @@ pub mod runtime;
 pub mod timer;
 pub mod util;
 
-pub use executor::spawn;
+pub use executor::{
+    spawn,
+    spawn_handle,
+    SpawnHandle,
```

Comment:
I re-exported them because otherwise the relative links in the RustDoc for `spawn_handle` broke. If that isn't important, I can remove the re-exports.